tests/python/pytest_framework.py

```diff
@@ -74,4 +74,11 @@ def __init__(self, opinfos, *, scope=None):
 
         atexit_serde_check()
 
-        create_op_test.__init__(self, opinfos, scope=scope)
+        # Replicate create_op_test.__init__ to have correct scope
```

Comment:
to make this slightly more future proof. should we still be calling `super().__init__` but with the updated `scope` arg instead?

Reply:
admittedly, it doesn't make much of a difference given the code as-is.

Reply:
But that adds to the scope if you call another function.

Reply:
sorry, I meant that we keep the part about resolving the scope here, but just pass the resolved scope. So we get to re-use the `self.opinfos = opinfos` in the super methods.

Again, this is really just nitpicking... It's your call :laughing:

Reply:
Since I needed to rebase, I added this change.